Z64Utils/Forms/ObjectAnalyzerForm.cs

```diff
@@ -299,7 +498,9 @@ private void listView_map_SelectedIndexChanged(object sender, EventArgs e)
                         textBox_holderInfo.Text = sw.ToString();
                         break;
                     }
-                case Z64Object.EntryType.SkeletonLimb:
+                case Z64Object.EntryType.StandardLimb:
+                case Z64Object.EntryType.LODLimb:
+                case Z64Object.EntryType.SkinLimb:

```

Comment:
I made them the same Holder in the hopes that it will make adding the other limb types to the renderer easier. I don't think separate holders are any more beneficial in themselves either.

Reply:
well my complaint is more about SkeletonLimbHolder I guess ( https://github.com/Thar0/Z64Utils/blob/more-detections/Z64Utils/Z64/Z64Object.cs#L232 )

```cs
            // Standard and LOD Limb Only
            public SegmentedAddress DListSeg { get; set; }
            // LOD Limb Only
            public SegmentedAddress DListFarSeg { get; set; }
            // Skin Limb Only
            public int SegmentType { get; set; } // indicates the type of data pointed to by SkinSeg
            public SegmentedAddress SkinSeg { get; set; }
```

this is kind of bad when working in an OOP context